code/drasil-lang/lib/Language/Drasil/Document/Contents.hs

```diff
@@ -38,7 +33,7 @@ unlbldExpr :: ModelExpr -> Contents
 unlbldExpr c = UlC $ ulcc $ EqnBlock c
 
 -- | Unlabelled code expression
-unlbldCode :: CodeExpr -> Contents
+unlbldCode :: Expr -> Contents
```

Comment:
It's definitely a side effect of our lack of tests that you can get away with this! If this were not a partial change towards an improved state, I'd say 'no', But I understand that this is temporary.